src/client/voice/receiver/PacketHandler.js

```diff
@@ -78,6 +79,23 @@ class PacketHandler extends EventEmitter {
     const ssrc = buffer.readUInt32BE(8);
     const user = this.userFromSSRC(ssrc);
     if (!user) return;
+
+    if (this.speakingTimeouts.has(ssrc)) {
+      clearTimeout(this.speakingTimeouts.get(ssrc));
+      this.speakingTimeouts.delete(ssrc);
+    } else {
+      this.connection.onSpeaking({ user_id: user.id, ssrc: ssrc, speaking: 1 });
+    }
+    const speakingTimer = setTimeout(() => {
+      try {
+        this.connection.onSpeaking({ user_id: user.id, ssrc: ssrc, speaking: 0 });
+        this.speakingTimeouts.delete(ssrc);
+      } catch (ex) {
+        // Connection already closed, ignore
+      }
+    }, 50);
```

Comment:
What does this `50` represent? Magic numbers should be documented.

Reply:
Good going, done and committed.